[Transaction] Support transactional Id expiration (#895)
streamnative/kop: docs/configuration.md

| brokerId                     | The broker ID that is used to create the producer ID.  | 1       |
| txnLogTopicNumPartitions     | the number of partitions for the transaction log topic. | 50      |
| txnAbortTimedOutTransactionCleanupIntervalMs | The interval in milliseconds at which to rollback transactions that have timed out. | 10000 |

> | transactionalIdExpirationMs | The time in ms that the transaction coordinator will wait without receiving any transaction status updates for the current transaction before expiring its transactional id. | 604800 |

Reviewer: ```suggestion
| transactionalIdExpirationMs | The time (in ms) that the transaction coordinator waits without receiving any transaction status updates for the current transaction before expiring its transactional ID. | 604800 |
```

> | transactionsRemoveExpiredTransactionalIdCleanupIntervalMs | The interval in milliseconds at which to remove transactions that have expired. | 3600 |

Reviewer: ```suggestion
| transactionsRemoveExpiredTransactionalIdCleanupIntervalMs | The interval (in ms) at which to remove expired transactions. | 3600 |
```

> | enableTransactionalIdExpiration | Whether to enable transactional id expiration. | true |

Reviewer: ```suggestion
| enableTransactionalIdExpiration | Whether to enable transactional ID expiration. | true |
```
